parser/tidb/tidb_parser_test.go

```diff
@@ -0,0 +1,23 @@
+package tidb
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTiDBParser(t *testing.T) {
+	a := assert.New(t)
+
+	e, i := Parse("select * from tbl where id = 1")
```

Comment:
Maybe add test case for wrong standard SQL statement and wrong SQLFlow statement to test if the parser can catch expected error in these cases?